### PR TITLE
Use latest components + accessibility fixes

### PR DESCRIPTION
**Pull request by skjnldsv**
Capture d’écran_2022-04-13_08-24-11

### CODE REVIEW

File: src/AdminSettings.vue [hunk 1]
```diff
@@ -29,85 +29,73 @@
 
 		<ul class="password-policy__settings-list">
 			<li>
-				<input id="password-policy__settings__min-length"
-					v-model="config.minLength"
-					type="number"
-					@change="updateNumberSetting('minLength')">
-				<span>{{ t('password_policy', 'Minimum password length') }}</span>
+				<label>
+					<input id="password-policy__settings__min-length"
```
Comment (jotoeri) [inline src/AdminSettings.vue:33]: Just researched for a moment, as i didn't know the implicit label yet. I just knew the explicit version with `<label for="">`.
Thus i just wanted to mention, that i found this comment on the internet... 🤷‍♂️ 
>_Generally, explicit labels are better supported by assistive technology._
https://www.w3.org/WAI/tutorials/forms/labels/

File: src/AdminSettings.vue [hunk 2]
```diff
@@ -29,85 +29,73 @@
...
 		<ul class="password-policy__settings-list">
-			<li />
 			<li>
-				<input id="password-policy__settings__enforce-non-common"
-					v-model="config.enforceNonCommonPassword"
-					type="checkbox"
-					class="checkbox"
-					@change="updateBoolSetting('enforceNonCommonPassword')">
-				<label for="password-policy__settings__enforce-non-common">
+				<CheckboxRadioSwitch :checked.sync="config.enforceNonCommonPassword"
```
Comment (jotoeri) [inline src/AdminSettings.vue:71]: How about using type `switch`? Esp. as there is no save-feedback on these Checkboxes? 🤔

File: src/AdminSettings.vue (hunk 1 above)
Comment (jotoeri) [inline src/AdminSettings.vue:33]: Wanna add a `min="0"` ? Just found this when testing by keyboard. 😉

Comment (skjnldsv) [reply src/AdminSettings.vue:33]: Fair enough :wink:

File: src/AdminSettings.vue (hunk 2 above)
Comment (skjnldsv) [reply src/AdminSettings.vue:71]: Good idea!

**Comment by skjnldsv** [comment]
Done @jotoeri, looks nice!!
Thanks for the help :hugs:

**Review by nickvergessen — APPROVED** [review]

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**